packages/launcher/src/renderer/UIComponents/CopyableBlock.js

```diff
@@ -0,0 +1,64 @@
+//@flow
+
+import React, { Component } from 'react'
+import { Text, Button } from '@morpheus-ui/core'
+import { CopyToClipboard } from 'react-copy-to-clipboard'
```

Comment:
Rather than adding an extra library could you use [Electron's clipboard API](https://electronjs.org/docs/api/clipboard) for this please?